src/Synergism.ts

```diff
@@ -1119,6 +1120,9 @@ const loadSynergy = (reset = false) => {
         for (let j = 1; j <= 50; j++) {
             updateCubeUpgradeBG(j);
         }
+        for (let j = 1; j <= 20; j++) {
```

Comment:
```suggestion
        const platUpg = document.querySelectorAll('img[id^="platUpg"]');
        for (let j = 1; j <= platUpg.length; j++) {
```

that way other elements can be added in the future without having to change the hardcoded number of elements